src/CourseDashboard.cpp

```diff
@@ -82,3 +83,55 @@ void CourseDashboard::updateUser(User & user)
         break;
     }
 }
+
+void CourseDashboard::saveToFile(User & user)
+{
+    std::fstream file;
+    try {
+    file.open("../src/UsersList.txt",std::ios::app);
+    file << user.getAllInfoToFile() << std::endl;
+    file.close();
```

Comment:
formatting - indentations
You don't beed to call `close()`. fstream implements RAII, it closes automatically in destructor.

Reply:
Corrected - close function removed.